fuzz/Cargo.toml

```diff
@@ -15,7 +15,7 @@ honggfuzz_fuzz = ["honggfuzz"]
 honggfuzz = { version = "0.5", optional = true }
 afl = { version = "0.8", optional = true }
 regex = { version = "1.4"}
-miniscript = { path = "..", features = ["fuzztarget", "compiler"] }
```

Comment:
Why are we changing this? The fuzzer won't be able to come up with hash preimages or signatures without the fuzztarget on.

Reply:
What does the `fuzztarget` feature do in rust-bitcoin? I think the only use we have in rust-miniscript is pass through for `rust-bitcoin`.

Reply:
I think I disabled them because I was not able to come up with secp Signature required for satisfier.